centaur/src/main/resources/standardTestCases/wdl_draft3/draft3_infer_version/draft3_infer_version.wdl

```diff
@@ -1,6 +1,11 @@
 
 # Some heading commentary...
 
+# FIXME/TODO:
+#
+# WDL version metadata is casted to String somewhere between WDL parsing and metadata checking in Centaur.
+# As a temporary measure to make this test pass, this test is expecting the WDL version to be a String.
+# This is an issue in Centaur and does not affect production.
```

Comment:
This is weird but I think totally fine... it makes sense to me that the version would be a string, I don't think we need to work around that.